backend/data/src/main/kotlin/io/tolgee/service/notification/NotificationService.kt

```diff
@@ -0,0 +1,23 @@
+package io.tolgee.service.notification
+
+import io.tolgee.model.Notification
+import io.tolgee.repository.NotificationRepository
+import org.springframework.data.domain.Page
+import org.springframework.data.domain.Pageable
+import org.springframework.stereotype.Service
+
+@Service
+class NotificationService(
+  private val notificationRepository: NotificationRepository,
+) {
+  fun getNotifications(
+    userId: Long,
+    pageable: Pageable,
+  ): Page<Notification> {
+    return notificationRepository.fetchNotificationsByUserId(userId, pageable)
+  }
+
+  fun createNotification(notification: Notification) {
```

Comment:
This should be named `save`, create usually transforms Dto to the entity and saves it or does some advanced logic.